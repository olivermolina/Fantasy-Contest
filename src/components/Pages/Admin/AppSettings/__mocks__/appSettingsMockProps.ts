export const appSettingsMockProps = {
  appSettings: {
    BONUS_CREDIT_FREE_ENTRY_EQUIVALENT: '25',
    STAKE_TYPE_FREE_ENTRY: 'ALL_IN,INSURED',
    NUMBER_OF_PLAYERS_FREE_ENTRY: '2,3,4',
    MAX_MATCH_DEPOSIT_AMOUNT: '100',
    DEPOSIT_AMOUNT_OPTIONS: '10,25,50,75,100,200',
    RELOAD_BONUS_TYPE: 'FLAT',
    RELOAD_BONUS_AMOUNT: '0',
    MAX_RETENTION_BONUS: '50',
    RETENTION_BONUS_WEEKLY_CHANCE: '1',
    RETENTION_BONUS_MATCH_MULTIPLIER: '1',
    REFERRAL_CREDIT_AMOUNT: '25',
    REFERRAL_CUSTOM_TEXT:
      'Refer a friend from the link or code in your profile and get $25 in bonus credit!!',
    MIN_BET_AMOUNT: '1',
    MAX_BET_AMOUNT: '50',
    MAX_MARKET_ODDS: '150',
    MIN_MARKET_ODDS: '-150',
    initial: false,
    CHALLENGE_PROMO_MESSAGE:
      'Pick 2-4 players. Predict if they will get MORE or LESS than their projection.',
    REPEAT_ENTRIES_LIMIT: '1',
  },
  onSubmit: () => console.log('onSubmit'),
};
