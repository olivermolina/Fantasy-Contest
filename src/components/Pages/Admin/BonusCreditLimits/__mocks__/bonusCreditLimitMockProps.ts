import { BetStakeType } from '@prisma/client';

export const bonusCreditLimitMockProps = {
  onSubmit: () => console.log('submit'),
  data: {
    signupFreeEntry: false,
    bonusCreditFreeEntryEquivalent: 50,
    bonusCreditLimits: [
      {
        contestCategoryId: '1',
        numberOfPicks: 2,
        id: '30387acd-660c-46f0-a9e6-af87c007e6fb',
        enabled: true,
        stakeTypeOptions: [BetStakeType.ALL_IN, BetStakeType.INSURED],
        bonusCreditFreeEntryEquivalent: 25,
      },
      {
        contestCategoryId: '2',
        numberOfPicks: 3,
        id: '316555e5-b447-468a-b57f-381bbf64919a',
        enabled: true,
        stakeTypeOptions: [BetStakeType.ALL_IN, BetStakeType.INSURED],
        bonusCreditFreeEntryEquivalent: 50,
      },
      {
        contestCategoryId: '3',
        numberOfPicks: 4,
        id: 'fc063fcd-1a25-46e2-9361-d947eec1f9a9',
        enabled: true,
        stakeTypeOptions: [BetStakeType.ALL_IN, BetStakeType.INSURED],
        bonusCreditFreeEntryEquivalent: 40,
      },
      {
        contestCategoryId: '4',
        numberOfPicks: 5,
        id: 'a2542740-e90f-4763-8a3a-f7afba68898c',
        enabled: true,
        stakeTypeOptions: [BetStakeType.ALL_IN, BetStakeType.INSURED],
        bonusCreditFreeEntryEquivalent: 20,
      },
    ],
  },
  openAddUsersFreeEntryConfirmDialog: () => console.log('open'),
};
