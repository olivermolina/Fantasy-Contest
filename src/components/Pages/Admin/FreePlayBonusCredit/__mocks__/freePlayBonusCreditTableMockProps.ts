import { PaymentStatusCode } from '~/constants/PaymentStatusCode';
import { TransactionType } from '@prisma/client';

export const freePlayBonusCreditTableMockProps = {
  onSubmitDeletePlayFreeCredit: () => console.log('delete'),
  data: [
    {
      id: '1',
      transactionDate: new Date(),
      amountBonus: 100,
      status: PaymentStatusCode.COMPLETE,
      type: TransactionType.CREDIT,
    },
    {
      id: '2',
      transactionDate: new Date(),
      amountBonus: 100,
      status: PaymentStatusCode.CANCELLED,
      type: TransactionType.DEBIT,
    },
  ],
  totalBonusCreditBalance: 0,
};
