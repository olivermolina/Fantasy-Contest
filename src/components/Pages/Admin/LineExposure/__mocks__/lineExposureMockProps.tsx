import { League } from '@prisma/client';

export const lineExposureMockProps = {
  data: [
    {
      id: '1',
      name: 'LeBron James',
      category: 'Points',
      league: League.NBA,
      status: 'Scheduled',
      matchup: 'LAL @ DEN',
      total: 35,
      gamedate: '2023-01-23',
      gametime: '03:00:00 PM',
      underOdds: 100,
      overOdds: -120,
      overExposure: 4,
      underExposure: 2,
      betLegs: [
        {
          id: 'ticket-id',
          betId: 'bet-id',
          entryDate: '2023-01-22',
          payout: 100,
          stake: 10,
          type: 'OVER_ODDS',
          user: {
            id: 'user-1',
            firstName: 'John',
            lastName: 'Doe',
            username: 'jdoe',
          },
          status: 'WIN',
        },
      ],
    },
  ],
  setDateRange: () => console.log('setDateRange'),
  dateRange: null,
  selectedLeague: League.NBA,
  setSelectedLeague: () => console.log('setSelectedLeague'),
};
