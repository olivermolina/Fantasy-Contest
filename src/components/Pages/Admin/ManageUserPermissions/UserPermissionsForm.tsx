import React, { useEffect } from 'react';
import { useFieldArray, useForm } from 'react-hook-form';
import { zodResolver } from '@hookform/resolvers/zod';
import * as z from 'zod';
import type { AdminModuleByCategoryType } from '~/server/routers/admin/getModulePermissions';
import ModuleUserPermissions from './ModuleUserPermissions';
import Button from '@mui/material/Button';
import { UserPermissionFormValidationSchema } from '~/server/routers/admin/saveModulePermissions';

export type UserPermissionFormInputs = z.infer<
  typeof UserPermissionFormValidationSchema
>;

interface Props {
  userId: string;
  userPermissions: AdminModuleByCategoryType[];
  onSubmit: (data: UserPermissionFormInputs) => void;
  handleClose: () => void;
}

export default function UserPermissionsForm(props: Props) {
  const { userId, userPermissions, handleClose } = props;
  const { handleSubmit, control, reset } = useForm<UserPermissionFormInputs>({
    resolver: zodResolver(UserPermissionFormValidationSchema),
    defaultValues: {
      userId,
      userPermissions,
    },
  });
  const { fields } = useFieldArray({
    control,
    name: 'userPermissions',
  });

  useEffect(() => {
    reset({
      userId,
      userPermissions,
    });
  }, [userId, userPermissions]);

  return (
    <form onSubmit={handleSubmit(props.onSubmit)}>
      <div className={'flex flex-col gap-4'}>
        {fields.map((field, index) => (
          <ModuleUserPermissions
            key={field.id}
            control={control}
            fieldIndex={index}
            userModule={{
              modulePermissions: field.modulePermissions,
              category: field.category,
            }}
          />
        ))}
        <div className={'flex justify-center gap-2 p-4'}>
          <Button variant={'outlined'} onClick={handleClose} sx={{ width: 75 }}>
            Cancel
          </Button>
          <Button type={'submit'} variant={'contained'} sx={{ width: 75 }}>
            Save
          </Button>
        </div>
      </div>
    </form>
  );
}
