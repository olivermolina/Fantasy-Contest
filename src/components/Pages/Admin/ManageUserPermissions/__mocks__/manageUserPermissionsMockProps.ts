import { UserStatus } from '@prisma/client';

export const manageUserPermissionsMockProps = {
  subAdmin: {
    id: '1',
    username: 'Employee 1',
    status: UserStatus.ACTIVE,
  },
  agents: [
    {
      id: '1',
      username: 'agent1',
      status: UserStatus.ACTIVE,
    },
    {
      id: '2',
      username: 'agent2',
      status: UserStatus.ACTIVE,
    },
    {
      id: '3',
      username: 'agent3',
      status: UserStatus.INACTIVE,
    },
    {
      id: '4',
      username: 'agent4',
      status: UserStatus.INACTIVE,
    },
  ],
  handleOpenManagePermission: () => console.log('handleOpenManagePermission'),
};
