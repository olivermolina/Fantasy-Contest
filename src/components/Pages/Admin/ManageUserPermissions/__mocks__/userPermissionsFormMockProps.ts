export const userPermissionsFormMockProps = {
  userId: 'userId1',
  userPermissions: [
    {
      category: 'Management',
      modulePermissions: [
        {
          moduleId: '1',
          label: 'Insert bonus credit',
          checked: true,
        },
        {
          moduleId: '2',
          label: 'Manually enter offer and lines',
          checked: true,
        },
        {
          moduleId: '3',
          label: 'Balance by User ID',
          checked: false,
        },
        {
          moduleId: '4',
          label: 'Update User Limits',
          checked: false,
        },
        {
          moduleId: '5',
          label: 'Withdrawal Offer',
          checked: true,
        },
      ],
    },
    {
      category: 'Figures',
      modulePermissions: [
        {
          moduleId: 'fig1',
          label: 'Weekly Balance',
          checked: true,
        },
        {
          moduleId: 'fig2',
          label: 'Line Exposure',
          checked: true,
        },
        {
          moduleId: 'fig3',
          label: 'Player Totals By Range',
          checked: false,
        },
      ],
    },
    {
      category: 'Action',
      modulePermissions: [
        {
          moduleId: 'act1',
          label: 'Delete picks',
          checked: true,
        },
      ],
    },
  ],
  onSubmit: (data: any) => console.log(data),
  handleClose: () => console.log('handleClose'),
};
