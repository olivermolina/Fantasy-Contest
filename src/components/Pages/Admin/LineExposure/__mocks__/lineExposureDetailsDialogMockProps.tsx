import { lineExposureMockProps } from './lineExposureMockProps';

export const lineExposureDetailsDialogMockProps = {
  row: lineExposureMockProps.data[0],
  open: false,
  handleClose: () => console.log(close),
};
