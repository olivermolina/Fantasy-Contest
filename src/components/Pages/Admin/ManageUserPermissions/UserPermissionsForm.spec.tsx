import { render } from '@testing-library/react';
import UserPermissionsForm from './UserPermissionsForm';
import { userPermissionsFormMockProps } from './__mocks__/userPermissionsFormMockProps';

describe('UserPermissionsForm', () => {
  it('should render correctly', () => {
    const { container } = render(
      <UserPermissionsForm {...userPermissionsFormMockProps} />,
    );
    expect(container).toMatchSnapshot();
  });
});
