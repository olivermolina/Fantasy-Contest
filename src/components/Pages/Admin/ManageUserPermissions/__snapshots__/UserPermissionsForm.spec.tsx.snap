// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserPermissionsForm should render correctly 1`] = `
<div>
  <form>
    <div
      class="flex flex-col gap-4"
    >
      <div
        class="flex flex-col"
      >
        <p
          class="py-2 px-6 bg-selected text-white font-semibold"
        >
          Management
        </p>
        <div
          class="grid grid-cols-3 divide-x divide-y content-center border"
        >
          <div
            class="MuiFormGroup-root css-150xn9n-MuiFormGroup-root"
          >
            <label
              class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-j204z7-MuiFormControlLabel-root"
            >
              <span
                class="MuiButtonBase-root MuiCheckbox-root MuiCheckbox-colorPrimary PrivateSwitchBase-root MuiCheckbox-root MuiCheckbox-colorPrimary Mui-checked MuiCheckbox-root MuiCheckbox-colorPrimary css-12wnr2w-MuiButtonBase-root-MuiCheckbox-root"
              >
                <input
                  checked=""
                  class="PrivateSwitchBase-input css-1m9pwf3"
                  data-indeterminate="false"
                  type="checkbox"
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="CheckBoxIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.11 0 2-.9 2-2V5c0-1.1-.89-2-2-2zm-9 14l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </span>
              <span
                class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
              >
                Insert bonus credit
              </span>
            </label>
          </div>
          <div
            class="MuiFormGroup-root css-150xn9n-MuiFormGroup-root"
          >
            <label
              class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-j204z7-MuiFormControlLabel-root"
            >
              <span
                class="MuiButtonBase-root MuiCheckbox-root MuiCheckbox-colorPrimary PrivateSwitchBase-root MuiCheckbox-root MuiCheckbox-colorPrimary Mui-checked MuiCheckbox-root MuiCheckbox-colorPrimary css-12wnr2w-MuiButtonBase-root-MuiCheckbox-root"
              >
                <input
                  checked=""
                  class="PrivateSwitchBase-input css-1m9pwf3"
                  data-indeterminate="false"
                  type="checkbox"
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="CheckBoxIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.11 0 2-.9 2-2V5c0-1.1-.89-2-2-2zm-9 14l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </span>
              <span
                class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
              >
                Manually enter offer and lines
              </span>
            </label>
          </div>
          <div
            class="MuiFormGroup-root css-150xn9n-MuiFormGroup-root"
          >
            <label
              class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-j204z7-MuiFormControlLabel-root"
            >
              <span
                class="MuiButtonBase-root MuiCheckbox-root MuiCheckbox-colorPrimary PrivateSwitchBase-root MuiCheckbox-root MuiCheckbox-colorPrimary MuiCheckbox-root MuiCheckbox-colorPrimary css-12wnr2w-MuiButtonBase-root-MuiCheckbox-root"
              >
                <input
                  class="PrivateSwitchBase-input css-1m9pwf3"
                  data-indeterminate="false"
                  type="checkbox"
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="CheckBoxOutlineBlankIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </span>
              <span
                class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
              >
                Balance by User ID
              </span>
            </label>
          </div>
          <div
            class="MuiFormGroup-root css-150xn9n-MuiFormGroup-root"
          >
            <label
              class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-j204z7-MuiFormControlLabel-root"
            >
              <span
                class="MuiButtonBase-root MuiCheckbox-root MuiCheckbox-colorPrimary PrivateSwitchBase-root MuiCheckbox-root MuiCheckbox-colorPrimary MuiCheckbox-root MuiCheckbox-colorPrimary css-12wnr2w-MuiButtonBase-root-MuiCheckbox-root"
              >
                <input
                  class="PrivateSwitchBase-input css-1m9pwf3"
                  data-indeterminate="false"
                  type="checkbox"
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="CheckBoxOutlineBlankIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </span>
              <span
                class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
              >
                Update User Limits
              </span>
            </label>
          </div>
          <div
            class="MuiFormGroup-root css-150xn9n-MuiFormGroup-root"
          >
            <label
              class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-j204z7-MuiFormControlLabel-root"
            >
              <span
                class="MuiButtonBase-root MuiCheckbox-root MuiCheckbox-colorPrimary PrivateSwitchBase-root MuiCheckbox-root MuiCheckbox-colorPrimary Mui-checked MuiCheckbox-root MuiCheckbox-colorPrimary css-12wnr2w-MuiButtonBase-root-MuiCheckbox-root"
              >
                <input
                  checked=""
                  class="PrivateSwitchBase-input css-1m9pwf3"
                  data-indeterminate="false"
                  type="checkbox"
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="CheckBoxIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.11 0 2-.9 2-2V5c0-1.1-.89-2-2-2zm-9 14l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </span>
              <span
                class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
              >
                Withdrawal Offer
              </span>
            </label>
          </div>
        </div>
      </div>
      <div
        class="flex flex-col"
      >
        <p
          class="py-2 px-6 bg-selected text-white font-semibold"
        >
          Figures
        </p>
        <div
          class="grid grid-cols-3 divide-x divide-y content-center border"
        >
          <div
            class="MuiFormGroup-root css-150xn9n-MuiFormGroup-root"
          >
            <label
              class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-j204z7-MuiFormControlLabel-root"
            >
              <span
                class="MuiButtonBase-root MuiCheckbox-root MuiCheckbox-colorPrimary PrivateSwitchBase-root MuiCheckbox-root MuiCheckbox-colorPrimary Mui-checked MuiCheckbox-root MuiCheckbox-colorPrimary css-12wnr2w-MuiButtonBase-root-MuiCheckbox-root"
              >
                <input
                  checked=""
                  class="PrivateSwitchBase-input css-1m9pwf3"
                  data-indeterminate="false"
                  type="checkbox"
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="CheckBoxIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.11 0 2-.9 2-2V5c0-1.1-.89-2-2-2zm-9 14l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </span>
              <span
                class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
              >
                Weekly Balance
              </span>
            </label>
          </div>
          <div
            class="MuiFormGroup-root css-150xn9n-MuiFormGroup-root"
          >
            <label
              class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-j204z7-MuiFormControlLabel-root"
            >
              <span
                class="MuiButtonBase-root MuiCheckbox-root MuiCheckbox-colorPrimary PrivateSwitchBase-root MuiCheckbox-root MuiCheckbox-colorPrimary Mui-checked MuiCheckbox-root MuiCheckbox-colorPrimary css-12wnr2w-MuiButtonBase-root-MuiCheckbox-root"
              >
                <input
                  checked=""
                  class="PrivateSwitchBase-input css-1m9pwf3"
                  data-indeterminate="false"
                  type="checkbox"
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="CheckBoxIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.11 0 2-.9 2-2V5c0-1.1-.89-2-2-2zm-9 14l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </span>
              <span
                class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
              >
                Line Exposure
              </span>
            </label>
          </div>
          <div
            class="MuiFormGroup-root css-150xn9n-MuiFormGroup-root"
          >
            <label
              class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-j204z7-MuiFormControlLabel-root"
            >
              <span
                class="MuiButtonBase-root MuiCheckbox-root MuiCheckbox-colorPrimary PrivateSwitchBase-root MuiCheckbox-root MuiCheckbox-colorPrimary MuiCheckbox-root MuiCheckbox-colorPrimary css-12wnr2w-MuiButtonBase-root-MuiCheckbox-root"
              >
                <input
                  class="PrivateSwitchBase-input css-1m9pwf3"
                  data-indeterminate="false"
                  type="checkbox"
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="CheckBoxOutlineBlankIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </span>
              <span
                class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
              >
                Player Totals By Range
              </span>
            </label>
          </div>
        </div>
      </div>
      <div
        class="flex flex-col"
      >
        <p
          class="py-2 px-6 bg-selected text-white font-semibold"
        >
          Action
        </p>
        <div
          class="grid grid-cols-3 divide-x divide-y content-center border"
        >
          <div
            class="MuiFormGroup-root css-150xn9n-MuiFormGroup-root"
          >
            <label
              class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-j204z7-MuiFormControlLabel-root"
            >
              <span
                class="MuiButtonBase-root MuiCheckbox-root MuiCheckbox-colorPrimary PrivateSwitchBase-root MuiCheckbox-root MuiCheckbox-colorPrimary Mui-checked MuiCheckbox-root MuiCheckbox-colorPrimary css-12wnr2w-MuiButtonBase-root-MuiCheckbox-root"
              >
                <input
                  checked=""
                  class="PrivateSwitchBase-input css-1m9pwf3"
                  data-indeterminate="false"
                  type="checkbox"
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="CheckBoxIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.11 0 2-.9 2-2V5c0-1.1-.89-2-2-2zm-9 14l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </span>
              <span
                class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
              >
                Delete picks
              </span>
            </label>
          </div>
        </div>
      </div>
      <div
        class="flex justify-center gap-2 p-4"
      >
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-134h4fi-MuiButtonBase-root-MuiButton-root"
          tabindex="0"
          type="button"
        >
          Cancel
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium css-178vi6x-MuiButtonBase-root-MuiButton-root"
          tabindex="0"
          type="submit"
        >
          Save
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </form>
</div>
`;
